bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -102,34 +106,83 @@ public Builder setShortRpcTimeoutMs(int shortRpcTimeoutMs) {
      * The amount of milliseconds to wait before issuing a client side timeout for long RPCs.
      * @param longRpcTimeoutMs timeout value in milliseconds.
      * @return a {@link Builder} object, for chaining
+     *
+     * @deprecated Please use {@link #setMutateRpcTimeoutMs(int)} or {@link #setReadRowsRpcTimeoutMs(int)}.
      */
+    @Deprecated
     public Builder setLongRpcTimeoutMs(int longRpcTimeoutMs) {
       Preconditions.checkArgument(longRpcTimeoutMs > 0, "Long Timeout ms has to be greater than 0");
       this.longRpcTimeoutMs = longRpcTimeoutMs;
       return this;
     }
 
+    /**
+     * The amount of milliseconds to wait before issuing a client side timeout for mutation RPCs.
```

Comment:
you can say, " the amount of time in milliseconds", or " the number of milliseconds". "amount of milliseconds" is not a preferred way.